### PR TITLE
Survey fixes

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Fixed the word wrapping and changed title, subtitle, detail to all scroll. Strictly speaking, this is not how the scrolling works on device, but the UI/UX is much better than the scrollviews inside scrollviews that you get with a max lines.

Also fixed a few of the issues that were low-hanging fruit discovered on Wednesday.

https://sagebionetworks.jira.com/browse/DHP-1047
https://sagebionetworks.jira.com/browse/DHP-1048
https://sagebionetworks.jira.com/browse/DHP-1077
https://sagebionetworks.jira.com/browse/DHP-1076

### CODE REVIEW

**Review by Erin-Mounts — APPROVED** [review]
looks good

File: src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx [hunk 1]
```diff
@@ -143,25 +144,28 @@ function Factory(args: {
...
-      ) //<FreeText step={args.step} onChange={args.onChange} />
+            fontFamily: latoFont,
+            fontSize: '14px',
+            fontStyle: 'italic',
+            fontWeight: '400',
+            textAlign: 'left',
+          }}>
+          {(args.step as Question).inputItem?.placeholder}
+        </Box>
+      ) // Mobile devices do not support more than 250 characters of free text.
```
Comment (hallieswan) [inline src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx:157]: Could consider pulling this into its own component like the other subcontrols, e.g. something like TimeDuration.tsx or Numeric.tsx in src/components/surveys/survey-design/question-edit/phone-subcontrols

**Review by hallieswan — APPROVED** [review]
Looks good to me!

File: src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx (hunk 1 above)
Comment (syoung-smallwisdom) [reply src/components/surveys/survey-design/question-edit/QuestionEditPhone.tsx:157]: Done

**Pull request merged**